main.js

```diff
@@ -117,21 +118,25 @@ function renderAllItems(allItems) {
 }
 
 function renderApp(appObject) {
+  let $cartView = document.querySelector('.cart')
   let $view = document.querySelector('[data-view="' + appObject.view + '"]')
   if (appObject.view === 'catalog') {
     $view.innerHTML = ''
+    $cartView.innerHTML = ''
```

Comment:
You can move these duplicated lines out of your `if...else` if they should be run in either case.